ddl/foreign_key_test.go

```diff
@@ -48,26 +48,35 @@ func (s *testForeighKeySuite) TearDownSuite(c *C) {
 
 func testCreateForeignKey(c *C, ctx context.Context, d *ddl, dbInfo *model.DBInfo, tblInfo *model.TableInfo, fkName string, keys []string, refTable string, refKeys []string, onDelete ast.ReferOptionType, onUpdate ast.ReferOptionType) *model.Job {
```

Comment:
Using `(s *testForeighKeySuite) testCreateForeignKey(...)` can reduce the parameters.